src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -45,6 +45,26 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
+    /** Stores FoodItem in Datastore. */
+  public void storeFood(FoodItem food) {
+    Entity foodentity = new Entity("FoodItem", food.getID().toString());
```

Comment:
Can you please make this `foodEntity` instead of `foodentinity`. Consistent camel casing makes it much easier to read :)